docs/assistant/connect-to-openai.asciidoc

```diff
@@ -0,0 +1,72 @@
+[[assistant-connect-to-openai]]
+= Connect to OpenAI
+
+This page provides step-by-step instructions for setting up an OpenAI connector for the first time. This connector type enables you to leverage OpenAI's large language models (LLMs) within {kib}. You'll first need to create an OpenAI API key, then configure the connector in {kib}. 
+
+
+[discrete]
+== Configure OpenAI
+
+[discrete]
+=== Select a model
+
+Before creating an API key, you must choose a model. Refer to the https://platform.openai.com/docs/models/gpt-4-turbo-and-gpt-4[OpenAI docs] to select a model. Take note of the specific model name (for example `gpt-4-turbo`), you'll need it when configuring {kib}.
+
+NOTE: `GPT-4 Turbo` has the best performance. `GPT-4` and `GPT-3.5` are also supported. 
+
+[discrete]
+=== Create an API key
+
+To generate an API key:
+
+. Log in to the OpenAI platform and navigate to **API keys**.
+. Select **Create new secret key**.
+. Name your key, select an OpenAI project, and set the desired permissions.
+. Click **Create secret key** and then copy and securely store the key. It will not be accessible after you leave this screen.
+
+The following video demonstrates these steps.
```

Comment:
Super nitty, but: everything is so tiny in the video! Not a high priority to re-do now, but the next time you need to re-do this or similar videos, try resizing the window to minimize empty white space.